maxim/tests/test_portkey.py

```
@@ -0,0 +1,342 @@
+# Set up global logger state to debug for testing
+logging.basicConfig(level=logging.DEBUG)
+
+
+class TestPortkeyIntegration(unittest.TestCase):
+    def test_instrument_portkey_sync(self):
+        client = instrument_portkey(portkey_ai.Portkey(api_key=portkey_api_key,virtual_key=portkey_virtual_key),logger)        
+        self.assertIsInstance(client, MaximPortkeyClient)
+        response = client.chat.completions.create(messages=[{"role": "user", "content": "Tell me what is big bang theory in 100 words?"}], model="gpt-4o", max_tokens=1000)
+        print(response)
+
+    def test_portkey_tool_calls_sync(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_portkey_tool_calls_sync`

Add return type annotation: `None`

(ANN201)